public/locales/en/components.json

```
@@ -0,0 +1,135 @@
+{
+  "about": {
+    "modal-title": "About OpenSpace",
+    "img-alt-text": "OpenSpace logo",
```

```suggestion
    "logo-alt-text": "OpenSpace logo",
```
`img` is too vague here, and we might add another image in the future